src/main/java/org/qualityannotate/quality/sonarqube/client/SqIssue.java

```diff
@@ -24,10 +26,10 @@
  * @param textRange  E.g. startLine: 2, endline: 2, startOffset: 0, endOffset: 204
  */
 public record SqIssue(String key, String component, String project, String rule, String status, String resolution,
-                      @Deprecated String severity, String message, Integer lineNumber, String hash, String author,
-                      String effort, @Deprecated String type,
-                      org.qualityannotate.quality.sonarqube.client.SqIssue.TextRange textRange,
-                      boolean quickFixAvailable, List<Impact> impacts) {
+        @Deprecated String severity, String message, Integer lineNumber, String hash, String author, String effort,
+        @Deprecated String type, org.qualityannotate.quality.sonarqube.client.SqIssue.TextRange textRange,
+        boolean quickFixAvailable, List<Impact> impacts) {
+
```

Comment:
⚠️ maintainability: Add 'since' and/or 'forRemoval' arguments to the @Deprecated annotation. [details](https://sonarcloud.io/organizations/quyt/rules?open=java:S6355&rule_key=java:S6355)